src/re_frame/interceptor.cljc

```diff
@@ -1,7 +1,9 @@
 (ns re-frame.interceptor
   (:require
-    [re-frame.loggers :refer [console]]
-    [re-frame.interop :refer [ratom? empty-queue debug-enabled?]]))
+   [re-frame.loggers :refer [console]]
+   [re-frame.interop :refer [ratom? empty-queue debug-enabled?]]
+   [clojure.set :as set]
+   ))
```

Comment:
I think he means
it should be like the following at the end of the require.
 [clojure.set :as set]))